internal/ui/select.go

```diff
@@ -11,7 +12,7 @@ import (
 
 // Select is responsible for determining the specific UI function given select user option, the current platform
 // config values, and environment status (such as a TTY being present).
-func Select(verbose, quiet bool) UI {
+func Select(verbose, quiet bool, reportWriter io.Writer) UI {
```

Comment:
It'd be nice to explain the correct usage of this new parameter (even very briefly) in the function's doc comment